1-exercises/A-array-find/exercise.js

```diff
@@ -16,8 +16,11 @@ let names = [
   "Karim",
   "Ahmed",
 ];
+ //function longNameWithA(name) {
+ // return name.length > 7;
+//}
 
-let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+let longNameThatStartsWithA = names.find(name => name.length > 7);
```

Comment:
You have the right idea here but just need to also add logic for starting with A